packages/auth/amplify_auth_cognito_dart/lib/src/auth_plugin_impl.dart

```diff
@@ -90,8 +93,10 @@ class AmplifyAuthCognitoDart extends AuthPluginInterface<
   AmplifyAuthCognitoDart({
     SecureStorageInterface? credentialStorage,
     HostedUiPlatformFactory? hostedUiPlatformFactory,
+    EndpointInfoStoreManager? pinpointEndpointIdStore,
```

Comment:
Shouldn't be injected. We should use the dependency manager (state machine) for injecting in tests.